sound/soc/sof/pcm.c

```diff
@@ -707,7 +708,15 @@ static int sof_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd,
 		/* do nothing for HDA dai_link */
 		break;
 	case SOF_DAI_INTEL_ALH:
-		/* do nothing for ALH dai_link */
+		for_each_dpcm_fe(rtd, 0, dpcm) {
+			struct snd_soc_pcm_runtime *fe = dpcm->fe;
+
+			fe->dai_link->trigger[SNDRV_PCM_STREAM_PLAYBACK] =
+				SND_SOC_DPCM_TRIGGER_PRE;
+			fe->dai_link->trigger[SNDRV_PCM_STREAM_CAPTURE] =
+				SND_SOC_DPCM_TRIGGER_PRE;
```

Comment:
@RanderWang a question: this PR aims at reducing the "pop" noise for playback. Do we need to change capture too?

Reply:
@lyakh I just restore the default behavior, please check (https://github.com/thesofproject/linux/commit/fd274c2b7267bc5cff1777a462837572908b545c). I will communicate with QA to get more info.

Reply:
Will this be called multi times?

Reply:
I confirmed with QA, they didn't find noise in wav data.

Reply:
> Will this be called multi times?

just like hw_params

Reply:
@RanderWang just wondering why not do this in sof_link_load()?

Reply:
oh I guess we dont know if this is connected to an ALH BE during link_load.